.github/workflows/ci.yml

```
@@ -25,6 +25,8 @@ jobs:
     strategy:
       matrix:
         platform: [Ubuntu-20.04]
+        zeek-kafka-os: [ubuntu]
```

This suggestion should be applied once we iron out any "no space left on device" and Ubuntu build & issues.
```suggestion
        zeek-kafka-os: [centos, ubuntu]
```